### PR TITLE
Highlighted text: Destroy event bindings

### PR DESCRIPTION
**Pull request by lukaszkonieczny**
## Links

* https://wikia-inc.atlassian.net/browse/CE-3429
* https://wikia-inc.atlassian.net/browse/CE-3430

## Description

Destroy event bindings after transition to different page

## Reviewers

@Wikia/community-engineering @Wikia/x-wing

### CODE REVIEW

**Comment by kamilkoterba** [comment]
LGTM

**Comment by rogatty** [comment]
LGTM

File: front/main/app/components/article-content.js [hunk 1]
```diff
@@ -599,8 +600,7 @@ export default Ember.Component.extend(
 				selection = window.getSelection(),
 				range = document.createRange();
 
-			Ember.$(document).unbind('touchmove', this.debouncedScroll);
-			Ember.$(window).unbind('scroll', this.debouncedScroll);
+			this.destroyHighlightedEditorEvents();
```
Comment (vforge) [inline front/main/app/components/article-content.js]: Is there any reason to create a function here? Do we use it anywhere again?

**Comment by vforge** [comment]
LGTM, one minor question

File: front/main/app/components/article-content.js (hunk 1 above)
Comment (rogatty) [inline front/main/app/components/article-content.js]: It's also used in https://github.com/Wikia/mercury/pull/2129/files#diff-feada17d05a701229ac8e263399b8d1fR129

Comment (vforge) [inline front/main/app/components/article-content.js]: I'm missed that thanks!

Comment (lukaszkonieczny) [inline front/main/app/components/article-content.js]: Yes, we do :)

**Pull request closed without merging**